src/utils/awsAccess.ts

```diff
@@ -0,0 +1,89 @@
+import jwt from "@tsndr/cloudflare-worker-jwt";
+import { env } from "../typeDefinitions/default.types";
+import config from "../../config/config";
+import { discordTextResponse } from "./discordResponse";
+import { DISCORD_BASE_URL, AWS_IAM_SIGNIN_URL } from "../constants/urls";
+
+export async function processAWSAccessRequest(
+  discordUserId: string,
+  awsGroupId: string,
+  env: env,
+  channelId: number
+): Promise<void> {
+  const authToken = await jwt.sign(
+    { name: "Cloudflare Worker", exp: Math.floor(Date.now() / 1000) + 2 },
+    env.BOT_PRIVATE_KEY,
+    { algorithm: "RS256" }
+  );
+
+  try {
+    const base_url = config(env).RDS_BASE_API_URL;
+    const requestData = {
+      groupId: awsGroupId,
+      userId: discordUserId,
+    };
+
+    const url = `${base_url}/aws-access`;
```

Comment:
Please move the URL to a separate constant sir

Reply:
Also please name it more appropriately

Reply:
Have created as a constant in the same file as we can use the base URL depending on the env as per config